include/LeMonADE/updater/UpdaterAbstractCreate.h

```diff
@@ -252,6 +258,206 @@ bool UpdaterAbstractCreate<IngredientsType>::addMonomerInsideConnectedPair(uint3
   }
   return false;
 }
+/******************************************************************************/
+/**
+ * @brief function to add a new monomer two already existing ones.
```

Comment:
Can you add in this comment, that all monomers are supposed to be connected afterwards